clients/rancher/client.go

```diff
@@ -86,6 +90,25 @@ func NewClient(bearerToken string, session *session.Session) (*Client, error) {
 
 	c.Steve.Ops.Session = session
 
+	if rancherConfig.CLIVersion != "" {
+		projectConfig := &management.Project{
+			ClusterID: "local",
+			Name:      namegen.AppendRandomString("project"),
+		}
+
+		testProject, err := c.Management.Project.Create(projectConfig)
```

Comment:
Why are we creating a project? Creating a resource here seems off.

Reply:
The Rancher CLI `login` command by default uses an interactive method to choose which project you want to login to. To bypass this and login silently, you need to provide the project ID. That's why we have this here so that we're able to bypass this issue in the CLI.

Reply:
1. I don't think this belongs here it should be in the cli client itself.
2. Have a resource creation is more than I would like. Couldn't we use the default project?

Reply:
I can try and move this over to `cli.go`. But the resource creation is something I don't think we get around, regardless where it is. Especially with downstream clusters being a potential enhancement. 

The Rancher CLI gives options default and system projects. If you only have the local cluster, that could work, but that isn't realistic to what a user would be doing.

Reply:
Moved the project creation over to `cli.go`.

Reply:
Talked offline of this in a meeting. Agreed to leave this in